rallyrolebot/cogs/purchase_cog.py

```diff
@@ -0,0 +1,97 @@
+import json
+import sys
+import traceback
+
+import discord
+from discord.ext import commands, tasks
+from discord.utils import get
+
+import data
+import rally_api
+import validation
+import errors
+
+from cogs import update_cog
+
+from constants import *
+from utils import pretty_print
+
+
+class PurchaseCommands(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    async def cog_after_invoke(self, ctx):
+        await pretty_print(
+            ctx, "Command completed successfully!", title="Success", color=SUCCESS_COLOR
+        )
+
+    @errors.standard_error_handler
+    async def cog_command_error(self, ctx, error):
+        # All other Errors not returned come here. And we can just print the default TraceBack.
+        print("Ignoring exception in command {}:".format(ctx.command), file=sys.stderr)
+        traceback.print_exception(
+            type(error), error, error.__traceback__, file=sys.stderr
+        )
+
+    @commands.command(
+        name="purchase",
+        help="Display purchase message",
+    )
```

Comment:
Add validation or check decorator to make sure that this command is sent only in a guild
https://discordpy.readthedocs.io/en/latest/ext/commands/commands.html#checks

Reply:
Removed changes related to purchase and balance commands, already completed in another PR.